packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -367,29 +363,16 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		);
 	}
```

Comment:
the method above, onConnected, in called in container runtime, and seems to block the reconnection state of the container until all blobs are uploaded. this seems incorrect with the new behavior, specifically we should not wait for all blobs to be uploaded after we re-connect, as we no long shortciruit the upload on disconnect

Reply:
we should probably have an e2e test for this. i imagine we intercept the blob upload and create a promise we control. we then disconnect and reconnect, and ensure the container runtime doesn't block, e.g. can send ops, and after that we can resolve the upload promise we intercepted and it should complete.

Reply:
this still doesn't look right to me. i don't understand why we need this method at all.

Reply:
i would expect more changes in the container runtime, does this need to be called every time we connect? it doesn't seem like it. i would think we only need to block on stashed ops during the initial boot, and never again